samples/contrib/aws-samples/debugger_component_demo/debugger-component-demo.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+
+# Uncomment the apply(use_aws_secret()) below if you are not using OIDC
+# more info : https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples/README.md
+
+import kfp
+import json
+import os
+import copy
+from kfp import components
+from kfp import dsl
+from kfp.aws import use_aws_secret
+
+
+cur_file_dir = os.path.dirname(__file__)
+components_dir = os.path.join(cur_file_dir, '../../../../components/aws/sagemaker/')
+
+sagemaker_train_op = components.load_component_from_file(components_dir + '/train/component.yaml')
+
+debug_hook = {
+    'S3OutputPath':'s3://kubeflow-pipeline-data/mnist_kmeans_example/hook_config'
```

Comment:
Lets make the bucket name a constant that we format into the strings, and put that constant right at the top so it can easily be found and modified.